decoder/vaapidecoder_mpeg2.cpp

```diff
@@ -934,8 +886,7 @@ YamiStatus VaapiDecoderMPEG2::decode(VideoDecodeBuffer* buffer)
             if (m_previousStartCode
                 == YamiParser::MPEG2::MPEG2_SEQUENCE_HEADER_CODE) {
                 status = processConfigBuffer();
-                if (status != YAMI_SUCCESS
-                    && status != YAMI_DECODE_FORMAT_CHANGE) {
+                if (status != YAMI_SUCCESS) {
```

Comment:
don't modify this, there's no need to send the extra format change when sequence header code came.